tests/func/test_delta.py

```diff
@@ -11,20 +11,35 @@
 from datachain.lib.file import File, ImageFile
 
 
+def _get_short_ds_name(catalog, name, project_name, namespace_name) -> str:
```

Comment:
Why `_get_short_ds_name`? IMO it should be something like `_get_full_ds_name` 🤔

Reply:
it is trying to get the shortest name (check if namespace passed is default and project is default and doesn't use them)